### ISSUE TITLE
helm-service: Allow deployment to a specified branch

### ISSUE THREAD

**Issue by christian-kreuzberger-dtx** [keptn/keptn#4373]
Right now, helm-service can only be used to deploy into a namespace according to the scheme `$PROJECT-$STAGE`.

For remote execution plane scenarios this might be sub-optimal. For instance, I want to have three clusters where I deploy into the namespace `myapp`, and not `myapp-dev` / `myapp-hardening` / `myapp-production`.

I propose to lift this limitation and allow helm-service to deploy to any namespace, either defined in the GitOps repo or via an environment variable.

# Definition of Done

- [ ] helm-service can be configured such that artifacts are deployed to a namespace other than `$PROJECT-$STAGE`
- [ ] Documentation for continous-delivery and remote execution plane is updated

**Comment by thschue** [comment]
I think there should be multiple patterns for this. So for "our" use-case only the service-name should be enough ...

**Comment by jurgen-weber-deltatre** [comment]
yeah, can we please bring this back. This will be a deal breaker for us.
